.changeset/few-brooms-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/react': minor
+---
+
+side-nav: Add `always` and `whenActive` variants for sub-level items.
```

Comment:
I would elaborate on this more, especially since the default has changed

Reply:
- [ ] callout default change